docs/proposal-rose-suite-run.md

```diff
@@ -117,8 +123,8 @@ The remaining points can happen any time after 1) is complete although note that
     Everything else will be copied.
     * If needed we could make this configurable via a `.cylcignore` file (future
       enhancement).
-  * The installation should fail if `log`, share` or `work` exist in the source
-    directory.
+  * The installation should fail if `log`, share` or `work` exist in the
+    source directory.
```

Comment:
I assume `_cylc-install` should now be added to this list.
(also, note the missing "`" which is presumably my fault)

Reply:
It doesn't have to be in the list as the presence of a flow.cylc file would mark this as a workflow.

Reply:
The `_cylc-install` directory will be created in the same directory as the workflow if there is no run directory (`--no-run-name`) so we can't allow  a _cylc-install directory to be copied from the source.